chart/values.yaml

```diff
@@ -140,7 +141,7 @@ ucentralgw:
     # Kafka
     ucentral.kafka.enable: "false"
     ucentral.kafka.group.id: 1
-    ucentral.kafka.brokerlist: localhost:9092
+    ucentral.kafka.brokerlist: localhost:9092 # NOTE this must be changed
```

Comment:
Can't we set this to the URL of the Kafka broker that is included?

Reply:
We could, but since service name is generated based on release name we would have to use templates. Not sure if we want to add this or not?

Reply:
@dunaev-opsfleet the whole idea of the assembly is to allow us to install everything as one bundle. You can solve the issue you mentioned above pretty quickly - don't autogenerate the service names, just leave them as `kafka`. Every chart nowadays have an option to overwrite the autogenerated kafka name using values. This way, you will have a predictable name for kafka in every namespace the assembly will be deployed into. WDYT? @brennerm any other ideas?

Reply:
With the current configuration we deploy a Kafka broker that is not being used which is completely unexpected from a user perspective. Think we should default to the provided Kafka broker URL but give the user the option to overwrite it.

Reply:
OK, thanks for a feedback. Changed the values so that we could have a fixed service name